Cargo.lock

```diff
@@ -10,19 +10,19 @@ checksum = "8b5ace29ee3216de37c0546865ad08edef58b0f9e76838ed8959a84a990e58c5"
 
 [[package]]
 name = "ab_glyph"
-version = "0.2.30"
+version = "0.2.31"
```

Comment:
The Cargo.lock file changes contain a lot more than just an arrow update. Did you accidentally run `cargo update` instead of `cargo update -p arrow`?